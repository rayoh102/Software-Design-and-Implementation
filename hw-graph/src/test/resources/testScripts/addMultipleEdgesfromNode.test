# Tests multiple edge addition from one node

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Add another node
AddNode graph1 n2

# Add another node
AddNode graph1 n3

# Add edge
AddEdge graph1 n1 n2 e1

# Add another edge
AddEdge graph1 n1 n3 e2

# See if edge was added properly
ListChildren graph1 n1