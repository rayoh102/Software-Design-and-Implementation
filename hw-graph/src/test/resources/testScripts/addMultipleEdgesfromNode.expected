# Tests multiple edge addition from one node

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Add another node
added node n2 to graph1

# Add another node
added node n3 to graph1

# Add edge
added edge e1 from n1 to n2 in graph1

# Add another edge
added edge e2 from n1 to n3 in graph1

# See if edge was added properly
the children of n1 in graph1 are: n2(e1) n3(e2)